production/helm/loki/templates/config.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if not .Values.loki.existingSecretForConfig -}}
+apiVersion: v1
+{{- if eq .Values.loki.configStorageType "Secret" }}
+kind: Secret
+{{- else }}
+kind: ConfigMap
```

Comment:
Since this actually IS a ConfigMap, what is the rational behind not naming the file configmap.yaml?

Reply:
@JStickler It can become either a Secret or a ConfigMap, was also following the mimir chart setup linked in the issue which just lists it as `config` : https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/templates/mimir-config.yaml#L3-L7

Reply:
>It can become either a Secret or a ConfigMap,
I did actually read that in the description, but then when I saw `ConfigMap` in the yaml my brain just latched onto the fact that the file was a ConfigMap.  🤦  You're right, might as well make it a config file since it could be either one.